### DISCUSSION TITLE
Failed to update rose-pine/neovim

### DISCUSSION THREAD

**Discussion by ZichengWangdrzz** [rose-pine/neovim#162] [General]
I use rose-pine through packer.lua, do not have a colorscheme.lua, just using the basic setting. Did not modify what it was in the packer.lua. Rencently I added some other pulgins, and kept giving me error - Failed to update rose-pine/neovim. Here's code in my packer.lua directory.

`
use({
	  'rose-pine/neovim',
	  as = 'rose-pine',
	  config = function()
		  vim.cmd('colorscheme rose-pine')
	  end
  }) 
`

I noticed some updates of usage of repo, tried them, did not work as well. I will appreciate your help.

**Comment by mvllow** [comment]
This is my fault for force pushing — uninstalling and reinstalling should fix the error!

You could manually remove the folder or comment out your Rosé Pine packer code, `:PackerSync` to remove, uncomment and `:PackerSync` again.

Sorry for the inconvenience

**Reply by ZichengWangdrzz** [reply]
Fixed! no worries, thank you for the help :)
